Scalar.Common/Platforms/Mac/MacPlatform.cs

```diff
@@ -148,7 +151,32 @@ public override void IsServiceInstalledAndRunning(string name, out bool installe
 
         public override string GetTemplateHooksDirectory()
         {
-            return Path.Combine("/usr", "local", "git", ScalarConstants.InstalledGit.HookTemplateDir);
+            string gitExecPath = GitInstallation.GetInstalledGitBinPath();
+
+            // Resolve symlinks
+            if (NativeMethods.IsSymlink(gitExecPath))
+            {
+                string link = NativeMethods.ReadSymlink(gitExecPath);
+                if (link != null)
+                {
+                    if (Path.IsPathRooted(link))
+                    {
+                        gitExecPath = link;
+                    }
+                    else
+                    {
+                        string baseDir = Path.GetDirectoryName(gitExecPath);
+                        gitExecPath = Path.Combine(baseDir, link);
+                    }
+                }
+            }
+
+            // Compute the base installation path (../../)
+            string gitBinDir = Path.GetDirectoryName(gitExecPath);
+            string installBaseDir = Path.GetDirectoryName(gitBinDir);
+            installBaseDir = Path.GetFullPath(installBaseDir);
+
+            return Path.Combine(installBaseDir, ScalarConstants.InstalledGit.HookTemplateDir);
```

Comment:
Not a new issue, but should we respect [`init.templatedir`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-inittemplateDir)?

Reply:
We are going to the template directory directly to avoid user-space things changing the hook we install, so I'm not sure this is a good idea. If a user customized `init.templatedir`, then they likely have customized the contents (including deleting the hook). So, I'm not a fan of inspecting this config option.